src/library_wasmfs.js

```diff
@@ -247,13 +247,20 @@ FS.createPreloadedFile = FS_createPreloadedFile;
         __wasmfs_readdir_finish(state);
         return entries;
       });
-    }
+    },
     // TODO: mount
     // TODO: unmount
     // TODO: lookup
     // TODO: mknod
     // TODO: mkdev
     // TODO: rename
+    rename: (old_path, new_path) => {
+      return FS.handleError(withStackSave(() => {
+        var old_path_buffer = stringToUTF8OnStack(old_path);
+        var new_path_buffer = stringToUTF8OnStack(new_path);
```

Comment:
We're not super consistent about this, but let's use camelCase for JS locals, like here:

https://github.com/emscripten-core/emscripten/blob/c1098d659d54dbaac9354ff66f95310a86961238/src/library_wasmfs.js#L173

That is, `old_path_buffer => oldPathBuffer`.

I'd also do the same for the param, so `old_path => oldPath`.

(In comparison, we use `_wasmfs_rename` etc. with `_` because that's the C convention, and we are calling into a C API here.)